pkg/operators/clustermanager/controllers/clustermanagercontroller/clustermanager_crd_reconcile.go

```diff
@@ -146,10 +146,11 @@ func (c *crdReconcile) updateStoredVersion(ctx context.Context) error {
 		// Check migration status before update CRD stored version
 		// If CRD's StorageVersionMigration is not found, it means that the previous or the current release CRD doesn't need migration, and can contiue to update the CRD's stored version.
 		// If CRD's StorageVersionMigration is found and the status is success, it means that the current CRs were migrated successfully, and can contiue to update the CRD's stored version.
-		// Other cases, for example, the migration failed, we should not contiue to update the stored version, that will caused the stored old version CRs inconsistent with latest CRD.
+		// Other cases, for example, the migration failed or not aplied, we should not contiue to update the stored version, that will caused the stored old version CRs inconsistent with latest CRD.
 		svmStatus, err := migrationcontroller.IsStorageVersionMigrationSucceeded(c.hubMigrationClient, name)
-		if svmStatus == false && !errors.IsNotFound(err) {
-			return fmt.Errorf("failed to updateStoredVersion as StorageVersionMigrations %v: %v", name, err)
+		if !(svmStatus == true || errors.IsNotFound(err)) {
```

Comment:
I am confused about this `if`. If is true when `svmStatus == false && !errors.IsNotFound(err)`, why it is different from before?